### PR TITLE
Gangster Music

### PR DESCRIPTION
**Pull request by leonzh2k**
Test pull request

### CODE REVIEW

**Comment by lchua2314** [comment]
Awesome work!

**Comment by roundband5** [comment]
Good job

File: src/App.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ function App() {
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
         <p>
-          Edit <code>src/App.js</code> and save to reload.
+          Harbinger Musik
```
Comment (lchua2314) [inline src/App.js:10]: Spelled music wrong

Comment (leonzh2k) [reply src/App.js:10]: Will Edit

**Review by lchua2314 — APPROVED** [review]
Looks good!

**Review by roundband5 — APPROVED** [review]
Looks Good

**Pull request merged**